packages/react/src/components/Grid/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Grid from './Grid';
```

Comment:
```suggestion
```

I bet we could just use `export *` and drop the `import` and `export default`!

Reply:
This appears to disallow a default import for the user (`import Grid from './Grid';`)

Reply:
@zachhardesty7 I think that's totally fine internal to the project, the public way to consume this should be:

```js
import { Grid } from 'carbon-components-react';
```

Internally, we can just write it as:

```js
import { Grid, Row, Cell } from '../components/Grid';
```

Reply:
Whoops, of course the user would never import from `'./Grid'`... d12c3ef